schema-engine/connectors/schema-connector/src/warnings.rs

```diff
@@ -408,6 +410,12 @@ impl fmt::Display for Warnings {
             f,
         )?;
 
+        render_warnings(
+            "The following models have a JSON Schema defined in the database, which is not yet fully supported. Read more: https://pris.ly/d/todo",
+            &self.json_schema_defined,
+            f
+        )?;
```

Comment:
This adds the cli warning message

Reply:
Also, let's make sure we give the `pris.ly` link a proper URL. https://pris.ly/d/mongodb-jsonschema should be good enough